.circleci/s3_tfstate/main.tf

```diff
@@ -0,0 +1,13 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+
+resource "aws_s3_bucket" "refinebio-tfstate" {
+  bucket = "refinebio-tfstate"
```

Comment:
This should use the ${user} and ${stage} variables as bucket names are universally unique.